.github/workflows/test.yml

```diff
@@ -0,0 +1,61 @@
+name: Testing
+
+on:
+  push:
```

Comment:
Does this include testing when a new pull request is started? I saw that you had pull requests and push in there in [9f1a61d](https://github.com/Galvant/InstrumentKit/pull/304/commits/9f1a61d14eb510da0e6434c58c33118ee2666253), but now only push to any branch, so I'm wondering if the full test suite including coverage will run on a new pull request that is pushed from a fork.

I have in another project limited push testing to the `main` branch and added a test requirement for pull requests to `main` as:

```yaml
on:
  push:
    branches: [ main ]
  pull_request:
    branches: [ main ]
```